Formula/semgrep.rb

```diff
@@ -29,6 +30,15 @@ class Semgrep < Formula
   depends_on "pkg-config" => :build
   depends_on "pcre"
   depends_on "python@3.9"
+  depends_on "tree-sitter"
+
+  # To update, temporarily comment out this resource and run `brew update-python-resources semgrep`
+  # Then uncomment this resource and manually modify the commit hash based on
+  # https://github.com/returntocorp/semgrep/blob/#{tag}/scripts/install-tree-sitter-runtime
+  resource "ocaml-tree-sitter" do
```

Comment:
Not ideal for updating.

May be better to allow updating w/o commenting out and just carry over the non-pypi resources (perhaps with specific option for this, which can be added into `pypi_formula_mappings.json`).